### PR TITLE
fix: make addresses clickable

### PR DESCRIPTION
**Pull request by lynnshaoyu**
1) https://uniswaplabs.atlassian.net/browse/WEB-1628?atlOrigin=eyJpIjoiZGJjNTA5OTQ2ZWU1NGVlZGIxMTM5MzY0MzZiOTZhNzUiLCJwIjoiaiJ9
2) https://uniswaplabs.atlassian.net/browse/WEB-1775?atlOrigin=eyJpIjoiMjcyMjFhZWZmYzA0NDA5MmFjMDlkM2UyNWQzNWM1ODIiLCJwIjoiaiJ9

### CODE REVIEW

File: src/nft/components/collection/ActivityCells.tsx [hunk 1]
```diff
@@ -110,7 +110,14 @@ export const AddressCell = ({ address, desktopLBreakpoint }: AddressCellProps) =
       display={{ sm: 'none', xl: desktopLBreakpoint ? 'none' : 'flex', xxl: 'flex' }}
       className={styles.addressCell}
     >
-      <Box>{address ? shortenAddress(address, 2, 4) : '-'}</Box>
+      <a
+        target="_blank"
+        rel="noreferrer"
+        href={`https://etherscan.io/address/${address}`}
+        style={{ textDecoration: 'none' }}
```
Comment (evgenyboxer) [inline src/nft/components/collection/ActivityCells.tsx:117]: Shouldn't you use `ExternalLink` component here? and for the etherscan link there seems to be `getExplorerLink` available which seems the right one.

File: src/nft/components/collection/Activity.css.ts [hunk 2]
```diff
@@ -112,6 +112,23 @@ export const addressCell = style([
...
+  textDecoration: 'none',
+})
+
+globalStyle(`${addressCell} a[href]:hover`, {
+  color: `${themeVars.colors.textPrimary}`,
+  opacity: `${themeVars.opacity.hover}`,
+  textDecoration: 'none',
+})
+
+globalStyle(`${addressCell} a[href]:focus`, {
```
Comment (JackShort) [inline src/nft/components/collection/Activity.css.ts:126]: instead of global style can you just use a styled a link component (using styled-components)?

**Review by JackShort — CHANGES_REQUESTED** [review]

File: src/nft/components/collection/ActivityCells.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/nft/components/collection/ActivityCells.tsx:117]: thank you! great catch appreciate it.

**Review by JackShort — APPROVED** [review]
lgtm

**Pull request merged**